packages/expo/src/hooks/useSignInWithGoogle.types.ts

```diff
@@ -0,0 +1,14 @@
+import type { SetActive, SignInResource, SignUpResource } from '@clerk/shared/types';
+
+export type SignUpUnsafeMetadata = Record<string, unknown>;
```

Comment:
I'm pretty sure you don't need to declare this, as we augment the global scope to support the customer defining their own type for this. I think you can just reference `SignUpUnsafeMetadata` directly without any imports or anything.